instructors/templates/take_appointment.html

```diff
@@ -1,10 +1,12 @@
 {%extends "base.html"%} {% block content %}
 <div>
   <h1> Take Appointment </h1>
-  </br>
-  <form method="post" action="{% url 'take_appointment' instructor.pk %}">
+{% block media  %} {{ form.media }} {% endblock %}
+
+ <form method="post" action="{% url 'take_appointment' instructor.pk %}">
     {% csrf_token %} {{ form.as_p }} {{ formset.as_p }}
```

Comment:
not sure where formset came from? did u add it? because there is no variable named formset on branch dev
if u did .. explain what you are doing, if you can't then why do it without talking to me?